### PR TITLE
Bump Go version

### PR DESCRIPTION
**Pull request by prombot**

### CODE REVIEW

File: 1.17/Makefile.COMMON [hunk 1]
```diff
@@ -15,7 +15,7 @@ REPOSITORY    := quay.io/prometheus
 NAME          := golang-builder
 BRANCH        := $(shell git rev-parse --abbrev-ref HEAD)
 SUFFIX        ?= -$(subst /,-,$(BRANCH))
-VERSION       := 1.15.15
+VERSION       := 1.17.0
```
Comment (SuperQ) [inline 1.17/Makefile.COMMON:18]: Hmm, this should be `1.17`

Comment (simonpasquier) [reply 1.17/Makefile.COMMON:18]: Comparing to #114, I think it's fine. I've pushed a follow-up commit to fix the README though.

**Review by discordianfish — APPROVED** [review]

**Comment by pdf** [comment]
Can someone explain to me what the process is here? prombot just keeps closing and opening new PRs on loop, makes it very hard to track when this change goes live.

**Comment by SuperQ** [comment]
Yea, something is broken with the bot.

**Pull request closed without merging**